etc/checkfmt.sh

```diff
@@ -38,7 +38,7 @@ fi
 
 cd ..
 export PATH=$PATH:.
-FMT_OUT="$(flux fmt -c stdlib)"
+FMT_OUT="$(env GO111MODULE=on go run -tags '' ./cmd/flux/main.go fmt -c stdlib)"
 if [[ -n "$FMT_OUT" ]]; then
```

Comment:
You could also add `set -e` to the top of this script to ensure that it returns non-zero if any command returns an error.